WindowsServerDocs/networking/technologies/hcn/hcn-declaration-handles.md

```diff
@@ -143,19 +144,19 @@ HcnCloseNetwork(
 
 ## HCN_Endpoint
 
-A HCN Endpoint is an entity which is used to represent an IP endpoint on a HCN network and its associated system resources and policies. For example, a HCN endpoint will typically consist of a set of metadata (e.g. id, name, parent network id), its network identity (e.g., IP address, MAC address), and any endpoint specific policies to be applied (e.g. ACLs, routes).
+A HCN Endpoint is an entity which is used to represent an IP endpoint on an HCN network and its associated system resources and policies. For example, an HCN endpoint will typically consist of a set of metadata (for example, id, name, parent network id), its network identity (for example, IP address, MAC address), and any endpoint specific policies to be applied (for example, ACLs, routes).
```

Comment:
The sentence beginning for example, its quite complex. Are you able to split this up or simplify the sentence?